platform/public/js/cookieconsent-config.js

```diff
@@ -0,0 +1,64 @@
+import 'https://cdn.jsdelivr.net/gh/orestbida/cookieconsent@v3.0.0/dist/cookieconsent.umd.js';
+
+CookieConsent.run({
+   categories: {
+        necessary: {
+            enabled: true,  // this category is enabled by default
+            readOnly: true  // this category cannot be disabled
+        }
+    },
+
+    language: {
+        default: 'en',
+        translations: {
+            en: {
+                consentModal: {
+                    title: 'Cookies on this site',
+                    description: 'We use cookies to enhance and personalise your experience. Accept all cookies below, or select “Manage cookies” to view our Cookie Policy and see details of the cookies we use.',
+                    acceptAllBtn: 'Accept all',
+                    showPreferencesBtn: 'Manage cookies'
+                },
+                preferencesModal: {
+                    title: 'Manage cookie preferences',
+                    acceptNecessaryBtn: 'Reject all',
```

Comment:
Given there isn't anything to reject at the moment, perhaps better "Accept necessary cookies"?

Reply:
Renamed in 316009c6814d4a07c9e1143af388e6772723858f